app/server/utils/user.py

```diff
@@ -624,3 +625,19 @@ def change_initial_pin(user: User, new_pin):
 
 def change_current_pin(user: User, new_pin):
     change_pin(user, new_pin)
+
+
+def default_token(user: User) -> Token:
```

Comment:
i didn't follow the discussion on default org. do all users now have a default organisation? what's that organisation's token? trying to see if this method would always return something correct and/or if i can just go with one or the other of transfer account vs organisation

Reply:
I think u comment this before we chatted lol, but users won't necessarily have a default org, but there will necessarily be a system level organisation that can be found via 
`Organisation.query.filter_by(is_master=true)...`